catalog/content/core/content-core-api/src/main/java/ddf/content/util/Describable.java

```diff
@@ -21,7 +21,8 @@
  * @author Hugh Rodgers, Lockheed Martin
  * @author ddf.isgs@lmco.com
  */
-public interface Describable {
+@Deprecated
+public interface Describable extends org.codice.ddf.platform.api.Describable {
```

Comment:
@stustison said that would be a breaking API change - @michaelmenousek suggest we go this route to preserve the API while also enabling ours.